### ISSUE TITLE
ACF Pro should be optional

### ISSUE THREAD

**Issue by tacman** [wearemetabolism/boilerplate-symfony#1]
Instead of making ACF Pro a requirement, could you make it optional, the provide the instructions for adding it (via composer req).

I think the boilerplate demo should be able to run without ACFPro.

Thanks.

**Comment by wearemetabolism** [comment]
You are right, the boilerplate will be removed soon with the release of [wordpress-bundle 1.4](https://github.com/wearemetabolism/wordpress-bundle/tree/develo) and the new installation process make acf optional :), you can try it if you want, I will be happy to have your feedback before the public release
